wgsl/index.bs

```diff
@@ -6394,6 +6394,19 @@ TODO: Add links to the eventual memory model.
   </xmp>
 </div>
 
+## Value-steering functions ## {#value-steering-functions}
```

Comment:
I think we can figure out a larger class of functions that are related to communicating our intent to the compiler, but we can do it later (when the set of this functions is larger than 1).